addons/mail/tests/test_mail_activity_plan.py

```diff
@@ -0,0 +1,46 @@
+from odoo import Command
+from odoo.addons.mail.tests.test_mail_activity import ActivityScheduleCase
+from odoo.tests import tagged
+
+
+class ActivityPlanScheduleCase(ActivityScheduleCase):
```

Comment:
Don't think you need that class, TestActivityPlan is probably sufficient (why extra layers ?)

Reply:
@kgasior is importing this class for `cls.activity_type_todo`. We could also not import it and define `cls.activity_type_todo ` in `TestActivityPlan`. :thinking: